src/plugins/multiline/config.ts

```diff
@@ -0,0 +1,34 @@
+import { ConfigItem } from "plugins";
+
+export const configList = [
+  {
+    type: "number",
+    default: 300,
+    min: 0,
+    max: Infinity,
+    step: 0.001,
```

Comment:
0.001 step doesn't make sense here. 1 (maybe even 10) would make sense. Same for the delay below.

Reply:
Fixed! Yeah I'm not too sure why anyone would care about having those be that precise lol.